network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/link_button_block.html

```diff
@@ -1,5 +1,11 @@
 {% load wagtailcore_tags %}
 
-<div class="my-default">
-  <a class="btn {{ value.styling }}" href="{{ value.URL }}">{{ value.label }}</a>
+<div class="container {{block.block_type}}-block">
+  <div class="row justify-content-center">
+    <div class="streamfield-content">
+      <div>
```

Comment:
This `<div>` should either have a class attribute, or it should not be here and the `<a>` should have a class that makes it display:block.

Reply:
Updated in latest commit. Thanks!